python/test/unit/runtime/test_launch.py

```diff
@@ -96,72 +96,56 @@ def kernel(x):
         pass
 
     # launch kernel
-    triton.knobs.runtime.kernel_load_start_hook = hook_start
-    triton.knobs.runtime.kernel_load_end_hook = hook_end
+    triton.knobs.runtime.kernel_load_start_hook.add(hook_start)
+    triton.knobs.runtime.kernel_load_end_hook.add(hook_end)
     kernel[(1, 3, 2)](6)
     assert used_start_hook
     assert used_end_hook
     assert start_hash == end_hash
+    triton.knobs.runtime.kernel_load_start_hook.remove(hook_start)
+    triton.knobs.runtime.kernel_load_end_hook.remove(hook_end)
 
 
-# LATENCY_THRESHOLD_US = 46
```

Comment:
Since we have another kernel launch latency benchmark now. The code here can be removed